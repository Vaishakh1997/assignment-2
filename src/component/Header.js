import React, { Component } from 'react';
import '../App.css'
import { Route, BrowserRouter as Router, Link, Switch} from "react-router-dom";

class Header extends Component {
    state = {  }
    render() { 
        return ( 
            <div className="header">
                <Link style={{textDecoration:'none'}} to="/"><h1>Albums</h1></Link>
            </div>
         );
    }
}
 
export default Header;