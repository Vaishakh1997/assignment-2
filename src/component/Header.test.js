import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

let wrapped = shallow(<Header />);

describe('Header Component', () => {

    it('should render the Header Component correctly', () => {
        expect(wrapped).toMatchSnapshot();
    });

    it('includes link to Mission scene', () => {
        expect(wrapped.find('Link').props().to).toBe("/");
    });

    it('renders the Titles of Header', () => {
        expect(wrapped.find('h1').text()).toEqual("Albums");
    });

});