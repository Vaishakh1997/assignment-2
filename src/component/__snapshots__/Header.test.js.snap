// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should render the Header Component correctly 1`] = `ShallowWrapper {}`;
